docs/overview/about.md

```diff
@@ -6,43 +6,54 @@ description: A self-sovereign blockchain network that allows developers to expre
 
 # About Archway
 
-Archway is an incentivized smart contract platform that rewards developers — ushering in next gen dApps.
+Archway is an incentivized smart contract platform that rewards developers that build on the network with baked in incentives and rewards.
+ 
 
-The protocol gives developers the tools to quickly build and launch scalable cross-chain dApps and get rewarded for the value the dApps contribute to the network.
+The protocol gives developers the tools to quickly build and launch scalable cross-chain dapps and get rewarded for the value the dapps contribute to the network.
 
 ## The Archway Mission
 
-The overarching goal of Archway is to foster an array of diverse and sustainable applications by empowering and rewarding developers that build on the network with baked in incentives and rewards.
-
-The internet helped democratize access to information, but it hasn't fully democratized access to capital and value. Entrepreneurs and developers worldwide lack access to the same opportunities as their counterparts in more developed nations.
+The overarching goal of Archway is to foster an array of diverse and sustainable applications by empowering and rewarding developers that build on the network
 
-Archway aims to level the playing field. Built into the protocol itself, Archway rewards developers based on the value and impact of their dApp, rather than their close association or connections to capital. 
+The internet helped democratize access to information, but it hasn't fully democratized access to capital and value. The mission of Archway is to give access to entrepreneurs and developers worldwide who lack access to the same opportunities as their counterparts in more developed nations.
 
-Unlike existing Web 2.0 platforms or early versions of Web 3 Layer-1s, where tokens are mostly concentrated in the hands of the first few early participants, Archway is designed to be shared across all contributors and stakeholders. The entrepreneurs and developers who are building on Archway get a true stake in the growth and governance of the protocol itself.
+Built into the protocol itself, Archway rewards developers based on the value and impact of their dapp, rather based on someone's personal connections or close ties to funding. This will level the playing field for many who are locked out of the world of raising capital.  
```

Comment:
```suggestion
Built into the protocol, Archway rewards developers based on the value and impact of their dapp rather based on someone's connections or close ties to funding. This will level the playing field for many locked out of the world of raising capital.  
```